api/src/main/java/javax/enterprise/inject/spi/ProcessBeanAttributes.java

```diff
@@ -92,4 +92,18 @@
      * @throws IllegalStateException if called outside of the observer method invocation
      */
     public void veto();
+
+
+    /**
+     * Instructs the container to ignore all non-static, final methods with public, protected or default visibility declared on
+     * any bean type of the specific bean during validation of injection points that require proxyable bean type.
+     * These methods shouldn't be invoked on the bean instances.
+     *
+     * Calling this method will bypass standard rules for unproxyable bean types (section 3.11 of the spec)
```

Comment:
For me this sentence attempts to express the same as the first sentence above. I think it would be better to add the reference to the spec at the end of the first sentence, e.g. `... of injection points that require proxyable bean type, as defined in section 3.11 Unproxyable bean types`.

A better "warning sign" would be to move `These methods shouldn't be invoked on the bean instances.` to a separate paragraph and reword to something like:
`These method should never be invoked upon bean instances. Otherwise, unpredictable behavior results.`